PSDepend/PSDependMap.psd1

```diff
@@ -69,7 +69,7 @@
     PSGalleryNuget = @{
         Script = 'PSGalleryNuget.ps1'
         Description = 'Install a PowerShell module from the PowerShell Gallery without the PowerShellGet dependency'
-        Supports = 'windows'
+        Supports = 'windows', 'core', 'macos', 'linux'
```

Comment:
Hiyo!  Cool!  Does this run on mac/*nix now?  Just the change to `nuget` instead of `nuget.exe` did it?

Reply:
Yes, just the removal of the extension got it working on my machine. I successfully tested this code on OSX. While I have not tested this exact code on any other Nix, I have run some very similar work around code on Ubuntu as well.